README.md

```diff
@@ -4,111 +4,209 @@ An API for validating survey schemas.
+
+``` shell
+npm install
 ```
+
+### Install Poetry and Python dependencies
+
+``` shell
 curl -sSL https://install.python-poetry.org | python3 - --version 2.1.2
 poetry install
 ```
 
-## Running 
+## Running locally
+
+To run the app:
+``` shell
+make run
+```
+
+Validator runs on two ports, `5001` is the main validator app and `5002` is the Ajv validator.
+
+### Validator
 
-The `AJV_VALIDATOR_URL` defaults to `http://localhost:5002/validate`.
+Validator runs on `http://localhost:5001/validate`.
+
+If you need to change the port you can change the port variable in the Uvicorn settings in api.py:
+``` python
+uvicorn.run("api:app", workers=20, port=5001, reload=False)
+```
+If you want to run the app locally using multiple server workers you can also set reload to "False" here too.
+
+### Ajv Validator
+
+The Ajv validator defaults to running on `http://localhost:5002`.
 
 You can override this by setting the `AJV_VALIDATOR_SCHEME` , `AJV_VALIDATOR_HOST`, and `AJV_VALIDATOR_PORT` environment variables.
+The defaults for these are:
+- `AJV_VALIDATOR_SCHEME` = "http"
+- `AJV_VALIDATOR_HOST` = "localhost"
+- `AJV_VALIDATOR_PORT` = "5002"
 
 Alternatively, you can override the entire URL by setting the `AJV_VALIDATOR_URL` environment variable directly.
 
+### Running against a URL
 
-To run the app:
+Once validator is running, it can be called directly in the browser using the "/validate" endpoint and the "url" parameter for the address where the schema is located (eg. GitHub Gist raw json).
 
+As Validator runs on `localhost:5001` by default, here is an example of a command you can use to validate a schema via a URL:
+``` shell
+http://localhost:5001/validate?url=https://raw.githubusercontent.com/ONSdigital/eq-questionnaire-runner/refs/heads/main/schemas/test/en/test_address.json
 ```
-make run
-```
-If you want to run the app locally using multiple server workers set reload to "False" in the Uvicorn settings in api.py:
-```python
-uvicorn.run("api:app", workers=20, port=5001, reload=False)
-```
-The validator can be called directly in the browser using the "/validate" endpoint and the "url" parameter for the address where the schema is located (eg. GitHub Gist raw json). Only URLs from the ONSDigital organization are accepted.
+
+Only the following URLs and domains are accepted:
+- "https://gist.githubusercontent.com/"
+- "https://raw.githubusercontent.com/"
+- "onsdigital.uk"
+- "localhost"
+
+Also when using a URL from GitHub you can only validate schemas from the ONSdigital organisation and only against repos with the owner "ONSdigital".
+
+### Running against eQ Runner
+
+Also once you have validator running it can be used to run against eQ runner (`https://github.com/ONSdigital/eq-questionnaire-runner`). If you have runner spun up you can run the `make validate-test-schemas` script from within the root of runner which will run validator on the test runner schemas.
+
+### Running the Ajv (server) version of validator
```

Comment:
Very minor, but might want to make this title with 4 `####` as it's related to the `Running against eQ Runner` section?

Reply:
Is it? I thought it would be a different and separate way of running validator?

Reply:
Yeah will do 👍, to be honest I'm not sure on the use case for running it in isolation so I do wonder what the need for this section is

Reply:
Sorry, this is a bit of a saga now 😆 In our Actions for Runner, we have to spin up both the AJV and the validator itself (via the docker-compose), so I think Pete is right in that we always need BOTH AJV and Validator running to run any validation. 

I couldn't spot any use cases for only needing the AJV validator to run in isolation so might have to say in one of the other sections that running AJV is mandatory instead?

Reply:
Yeah maybe that's what we need to say here 👍 Something like:

"Running `make run` will start up the both of services required for validation. However, if you want to start AJV individually, run:"